code/API_definitions/Edge-Application-Management.yaml

```diff
@@ -897,13 +897,19 @@ components:
         and the value of the Edge Cloud Provider
         object. This value is used to identify an Edge Cloud zone
         between Edge Clouds from different Edge Cloud Providers.
+      required:
+        - edgeCloudZoneId
+        - edgeCloudZoneName
```

Comment:
How is uniqueness defined here? Is edgeCloudZoneId or edgeCloudZoneName are unique or their combination is?

Reply:
SimpleEdgeDiscovery has:

  * `edgeCloudZoneId` is a UUID for the Edge Cloud Zone.
  * `edgeCloudZoneName` is the common name of the closest Edge Cloud Zone to
  the user device.
  * `edgeCloudProvider` is the name of the operator or cloud provider of
  the Edge Cloud Zone.
  
  So, `edgeCloudZoneId` is expected to be unique, e.g. a namespaced URN

Reply:
I would like to point out that if edgeCloudZoneName (or edgeCloudZoneName + edgeCloudProvider) cannot uniquely identify the zone, i.e. only the UUID can uniquely identify the zone, then we won't be able to support a declarative API. That's probably ok if the API is mainly going to be accessed via a GUI by a human, but if we want to support automation and infra-as-code via yaml files, it would be much nicer to be able to have a declarative API.

Reply:
Would the edgeCloudZoneId be the only required parameter for the edgeCloudZone?